am-lib/src/main/java/uk/gov/hmcts/reform/amlib/AccessManagementService.java

```diff
@@ -112,7 +112,8 @@ public void grantExplicitResourceAccess(@NotNull @Valid ExplicitAccessGrant acce
      */
     @AuditLog("explicit access revoked by '{{mdc:caller}}' to resource '{{accessMetadata.resourceId}}' "
         + "defined as '{{accessMetadata.serviceName}}|{{accessMetadata.resourceType}}|{{accessMetadata.resourceName}}' "
-        + "from accessor '{{accessMetadata.accessorId}}': {{accessMetadata.attribute}}")
+        + "from accessor '{{accessMetadata.accessorId}}' with relationship {{accessMetadata.relationship}}: "
```

Comment:
Please wrap `{{accessMetadata.relationship}}` in single quotes for consistency. Unwritten rule is that all request arguments are in single quotes.

Reply:
👍